Add note clarifying that certain skipped tests are acceptable in upgrade (#428)
astronomer/docs: cloud/stable/04_customize-airflow/01_upgrade-to-airflow-2.md

1. Initialize the Airflow upgrade process via the Astronomer UI or CLI
2. Add the AC 1.10.15 image (Debian-only) to your `Dockerfile`:

    ```dockerfile
    FROM quay.io/astronomer/ap-airflow:1.10.15-buster-onbuild

>     ```

Reviewer: ```suggestion
    ```

```

Reviewer: Format for in-step code is 

[Step]
[Newline]
[Four spaces-indented code]
[Newline]
[Next step]

3. Deploy to Astronomer via `$ astro deploy`
    

>     For detailed guidelines on how to upgrade Airflow on Astronomer, read [Upgrade Airflow](https://www.astronomer.io/docs/cloud/stable/customize-airflow/manage-airflow-versions). For more information on 1.10.15, check out the [Airflow Release](https://github.com/apache/airflow/releases/tag/1.10.15) or the corresponding [AC 1.10.15 changelog](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md).

Reviewer: ```suggestion
For detailed guidelines on how to upgrade Airflow on Astronomer, read [Upgrade Airflow](https://www.astronomer.io/docs/cloud/stable/customize-airflow/manage-airflow-versions). For more information on 1.10.15, check out the [Airflow Release](https://github.com/apache/airflow/releases/tag/1.10.15) or the corresponding [AC 1.10.15 changelog](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md).
```
We don't need to indent here since it's not directly part of the listed steps.

>     > **Note:** Astronomer Certified exclusively supports a Debian-based Docker image for both Airflow 1.10.15 and 2.0. If you're currently running an Alpine-based AC image for an earlier version of Airflow, we recommend that you first test the corresponding Debian image for your current version. For example, if you're running an Alpine-based 1.10.12 image, try the Debian-based 1.10.12 image before you upgrade to 1.10.15.

Reviewer: ```suggestion
> **Note:** Astronomer Certified exclusively supports a Debian-based Docker image for both Airflow 1.10.15 and 2.0. If you're currently running an Alpine-based AC image for an earlier version of Airflow, we recommend that you first test the corresponding Debian image for your current version. For example, if you're running an Alpine-based 1.10.12 image, try the Debian-based 1.10.12 image before you upgrade to 1.10.15.
```

Don't think we need to indent the note here because it's not part of the steps

> > **Note:** In the upgrade check output above, you can ignore the `Fernet is enabled by default` failure, as that value is taken care of on Astronomer.

Reviewer: ```suggestion
> **Note:** In the upgrade check output above, you can ignore the `Fernet is enabled by default` and `Check versions of PostgreSQL, MySQL, and SQLite to ease upgrade to Airflow 2.0` tests. All of these details are handled by Astronomer. You can also ignore the and `Users must set a kubernetes.pod_template_file value` test even if you are running the Kubernetes Executor.
```

Reviewer: Since the note above seemed to be communicating the same general idea, I combined all ignorable tests into a single note. 
Side note: You only need to add a new line to notes when you introduce a new paragraph. Markdown will handle the formatting for the rest!

> > **Note:** You may see skipped tests for `Check versions of PostgreSQL, MySQL, and SQLite to ease upgrade to Airflow 2.0` and `Users must set a kubernetes.pod_template_file value`. These values are taken care

Reviewer: ```suggestion
```

> > of on Astronomer. You may ignore the kubernetes.pod_template_file test even if you are running a KubernetesExecutor.

Reviewer: ```suggestion
```

1. Initialize the Airflow upgrade process via the Astronomer UI or CLI
2. Add the AC 2.1.0 image (Debian-only) to your `Dockerfile`:

    ```dockerfile
    FROM quay.io/astronomer/ap-airflow:2.1.0-buster-onbuild

>     ```

Reviewer: ```suggestion
    ```

```
